sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedSource.cs

```diff
@@ -24,14 +24,16 @@ dataFeedDetail switch
                 AzureApplicationInsightsDataFeed d => new AzureApplicationInsightsDataFeedSource(d.DataSourceParameter),
```

Comment:
This file contains the transforms we do from public type to service type.
- Service => Public (used in Get, List, and the return of the Update operation).
- Public => Service (used in Create)
- Public => Service Patch (used in Update)